apps/emqx_rule_engine/src/emqx_rule_registry.erl

```diff
@@ -168,6 +170,14 @@ start_link() ->
 get_rules() ->
     get_all_records(?RULE_TAB).
 
+get_rules_order_by_ts() ->
```

Comment:
The function get_rules_ordered_by_ts/0 should be used both by API and CLI, and the test case for the API and CLI should be updated to verify the ordering.